src/main/java/com/swordglowsblue/artifice/api/builder/data/StateDataBuilder.java

```diff
@@ -3,11 +3,11 @@
 import com.google.gson.JsonObject;
 import com.swordglowsblue.artifice.api.builder.TypedJsonBuilder;
 
-public class BlockStateDataBuilder extends TypedJsonBuilder<JsonObject> {
```

Comment:
is this class something you added recently?

Reply:
yeah I think :/ .
It's also used by FluidState so I renamed it StateDataBuilder.